src/focus.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import css from 'styled-jsx/css'
+import { c_NEONS } from './theme'
+
+let count = 0
+
+const generateFocus = num => {
+  if (typeof num !== 'number') {
+    num = count++
+  }
+
+  return {
+    ...css.resolve`
+      button:focus,
+      *[href]:focus,
+      input:focus,
+      select:focus,
+      textarea:focus,
+      *[tabindex]:not([tabindex='-1']):focus {
+        position: relative;
+        display: inline-block;
+        outline: none;
+      }
+
+      button:focus::before,
+      *[href]:focus::before,
+      input:focus::before,
+      select:focus::before,
+      textarea:focus::before,
+      *[tabindex]:not([tabindex='-1']):focus::before {
+        border: 2px solid ${c_NEONS[num % c_NEONS.length]};
+        box-shadow: ${c_NEONS[num % c_NEONS.length]} 0px 0px 8px;
+        content: '';
+        position: absolute;
+        pointer-events: none;
+        inset: -2px;
```

Comment:
[`inset` has incredibly poor browser support.](https://developer.mozilla.org/en-US/docs/Web/CSS/inset#Browser_compatibility) Can you achieve this with a better supported css rule?

Reply:
That was just copied from the prototype website's css. I'll see if I can do it another way.

Reply:
Fixed.